api/v1/testkube.yaml

```diff
@@ -33,8 +33,210 @@ tags:
     description: "Executor operations"
   - name: webhook
     description: "Webhook operations"
+  - name: test-triggers
+    description: "Test Triggers CRUD operations"
+  - name: keymap
+    description: "Keymap for fields in a form"
 
 paths:
+  /keymap/triggers:
+    get:
+      tags:
+        - keymap
+        - api
+      summary: Test triggers keymap
+      description: Returns a keymap (supported/allowed fields) for the test trigger UI screen
+      operationId: getKeyMap
+      responses:
+        200:
+          description: successful get operation
+          content:
+            application/json:
+              schema:
+                type: object
+                items:
+                  $ref: "#/components/schemas/TestTriggerKeyMap"
+            text/yaml:
+              schema:
+                type: string
+  /triggers:
+    get:
+      tags:
+        - test-triggers
+        - api
+      summary: List test triggers
+      description: List test triggers from the kubernetes cluster
+      operationId: listTestTriggers
+      responses:
+        200:
+          description: successful list operation
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/TestTrigger"
+            text/yaml:
+              schema:
+                type: string
+        502:
+          description: problem communicating with kubernetes cluster
+          content:
+            application/problem+json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/Problem"
+
+    post:
+      tags:
+        - test-triggers
+        - api
+      summary: "Create new test trigger"
+      description: "Create new test trigger CRD inside a Kubernetes cluster"
+      operationId: createTestTrigger
+      requestBody:
+        description: test trigger body
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/"
```

Comment:
What schema?

Reply:
typo, should be `TestTriggerUpsertRequest`